app/input_objects/pages/long_lists_selection/bulk_options_input.rb

```diff
@@ -0,0 +1,49 @@
+class Pages::LongListsSelection::BulkOptionsInput < BaseInput
+  DEFAULT_OPTIONS = { selection_options: [],
+                      only_one_option: "true",
+                      include_none_of_the_above: false }.freeze
+
+  attr_accessor :include_none_of_the_above, :draft_question, :bulk_selection_options
+
+  validates :draft_question, presence: true
+  validates :include_none_of_the_above, inclusion: { in: %w[true false] }
+  validate :bulk_selection_options, :validate_selection_options
+
+  def assign_form_values
+    self.bulk_selection_options = draft_question.answer_settings[:selection_options].map { |option| option[:name] }.join("\n")
+    self.include_none_of_the_above = draft_question.is_optional
```

Comment:
Nice, this is a good way of initialising the input. I'll update the corresponding `options_input` controller to use this pattern.